pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/SortOperator.java

```diff
@@ -134,8 +150,10 @@ private void consumeInputBlocks() {
         for (Object[] row : container) {
           SelectionOperatorUtils.addToPriorityQueue(row, _rows, _numRowsToKeep);
         }
-
+        _operatorStats.endTimer();
```

Comment:
I noticed that in your output example `SORT` always reports zero input rows. Do you need to increment something here?